### PR TITLE
user-replenishments

### PR DESCRIPTION
**Pull request by RezenkovD**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 .idea/
 __pycache__/
 src/.env
+.vscode/settings.json
```
Comment (qqlexa) [inline .gitignore:4]: What about whole folder `.vscode/`?

File: src/models/status.py
Comment (qqlexa) [inline src/models/status.py:1]: Is it model status? I guess it can be moved to enum folder

File: src/routers/replenishment.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+def create_replenishments(
+    *,
+    db: Session = Depends(get_db),
+    current_user: User = Depends(get_current_user),
+    replenishments: CreateReplenishment,
+) -> ReplenishmentModel:
+    return services.create_replenishments(db, current_user.id, replenishments)
+
+
+@router.get("/all-time/", response_model=List[UserReplenishment])
```
Comment (qqlexa) [inline src/routers/replenishment.py:32]: `read_replenishments_all_time`, `read_replenishments_month` and `read_replenishments_time_range` have the same logic. Use query_params and only one route

**Review by qqlexa — CHANGES_REQUESTED** [review]

File: src/routers/replenishment.py (hunk 2 above)
Comment (RezenkovD) [reply src/routers/replenishment.py]: Thanks. This very good idea

**Review by qqlexa — APPROVED** [review]

**Pull request merged**